app/locales/en.js

```diff
@@ -9,7 +9,12 @@ export default {
     recents: 'Your recent classifications',
     talk: 'Talk',
     tutorialButton: 'Show the project tutorial',
-    miniCourseButton: 'Restart the project mini-course'
+    miniCourseButton: 'Restart the project mini-course',
```

Comment:
These should be added to the other language files, so that people know they're available to be translated.